test/libsolidity/semanticTests/abiEncoderV2/cleanup/function.sol

```diff
@@ -0,0 +1,28 @@
+pragma experimental ABIEncoderV2;
+
+contract C {
+	struct S { function() external f; }
+    function ggg(function() external x) external pure returns (uint256 r) {
+        assembly { r := calldataload(4) }
+    }
+    function h(S calldata x) external pure returns (uint256 r) {
+		x.f; // validation only happens here
+        assembly { r := calldataload(4) }
+    }
+	function dummy() external {}
+    function ffff(uint256 a) external view returns (uint256, uint256) {
+        S memory s = S(this.dummy);
+        assembly { mstore(s, a) }
+        return (this.ggg(s.f), this.h(s));
+    }
+}
+// ----
+// ffff(uint256): 0 -> 0, 0
```

Comment:
Hm... ``ffff(uint256)``, ``ggg(function)`` and ``h((function))`` are all 13 characters wide, so these should be nicely aligned as well - not sure why github messes that up here, since it displays it fine everywhere else...

Reply:
For some reason github puts `ff` in the space of a single character